forc/src/cli/mod.rs

```diff
@@ -26,9 +27,12 @@ struct Opt {
     /// the command to run
     #[clap(subcommand)]
     command: Forc,
+    /// Use verbose output.
+    #[clap(short, long, parse(from_occurrences), global = true)]
```

Comment:
This will have to change when we upgrade to clap v3.2:
https://github.com/clap-rs/clap/blob/v3.2.22/tests/derive/nested_subcommands.rs#L21